api/src/main/java/org/apache/iceberg/ManageSnapshots.java

```diff
@@ -39,7 +49,7 @@
    * @return this for method chaining
    * @throws IllegalArgumentException If the table has no snapshot with the given id
    */
-  Rollback toSnapshotId(long snapshotId);
+  ManageSnapshots rollback(long snapshotId);
```

Comment:
I think we should use `setCurrentSnapshot` instead of `rollback` because this doesn't need to check that the snapshot is an ancestor. Alternatively, we could have both and make the `rollbackTo` version check that the given snapshot _is_ an ancestor.

Reply:
done.